receiver/googlecloudspannerreceiver/internal/metadata/labelvalue.go

```diff
@@ -215,12 +215,26 @@ type lockRequest struct {
 
 func newLockRequestSliceLabelValue(metadata LabelValueMetadata, valueHolder interface{}) LabelValue {
 	value := *valueHolder.(*[]*lockRequest)
-	convertedValue := make([]string, len(value))
+	// During the specifics of this label we need to take into account only distinct values
+	uniqueValueItems := make(map[string]struct{})
+	var convertedValue []string
+
+	for _, valueItem := range value {
+		var valueItemString string
+		if valueItem.TransactionTag == "" {
+			valueItemString = fmt.Sprintf("{%v,%v}", valueItem.LockMode, valueItem.Column)
```

Comment:
You don't seem to have a test for this condition, do you?

Reply:
I've just updated test input data to cover this.